src/components/az-list/AZList.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./AZList.scss";
+
+import { testIds } from "../../constants";
+import { useAppPaths } from "../../hooks/routing";
+import { useStateValue } from "../../store/store";
+import { i18n } from "../../utils/i18n";
+
+const AZList = () => {
+    const { HomePath } = useAppPaths();
+    const [{ language }] = useStateValue();
+    const baseURL = `${HomePath()}?expand=`;
```

Comment:
This should change to be the correct url. Will update in a minute.

Reply:
```const baseURL = `${HomePath()}/expand=`;```?

Reply:
`useAppPaths` should be updated to contain:
`'Expand': '/expand/:expandChar'`
And then setting the URL should be done by calling `Expand(letter)`

Reply:
Got it! Thanks

Reply:
Updated